src/screens/UserManagement/UserRevamp/UserUtils.res

```diff
@@ -200,7 +200,7 @@ let getEntityType = valueDict => {
   let profileValue = valueDict->getOptionString("profile_value")
 
   switch (orgValue, merchantValue, profileValue) {
-  | (Some(_orgId), Some("all_merchants"), Some("all_profiles")) => "organisation"
+  | (Some(_orgId), Some("all_merchants"), Some("all_profiles")) => "organization"
```

Comment:
why this has been changed

Reply:
because api is expecting the `organization` instead of `organisation`